content/home/src/howto_guides/submit2nmdc.md

```diff
@@ -1,17 +1,17 @@
+## Introduction: Making metadata capture easy
+The NMDC has developed two integrated resources for improving metadata capture: the [NMDC Submission Portal](https://data.microbiomedata.org/submission/home) (released April, 2022) and the [NMDC Field Notes](https://microbiomedata.org/field-notes/) mobile app (released April 2025). Each resource was designed to lower barriers to capturing metadata and adhering to community standards, thereby addressing the critical gap of collecting necessary information describing a study, its biosamples, and associated assays.
 
 ### NMDC Submission Portal
-The NMDC Submission Portal was designed using a flexible framework leveraging a new modeling approach called the Linked Data Modeling Language (LinkML) and the template-driven spreadsheet tool, [DataHarmonizer](https://github.com/cidgoh/DataHarmonizer). It supports several different community standards, such as the _Minimal Information about any (x) Sequence_ (MIxS) standard from the Genomic Standards Consortium (GSC), the _PROV_ standard for provenance metadata, the _Proteomics Standards Initiative_ (PSI) standards for metaproteomics, and the _Metabolomics Standards Initiative_ (MSI) standards for metabolomics. The NMDC Submission Portal is an intuitive interface that allows researchers to provide information about their study, the metadata about the study’s biosamples, multi-omics data associated with the study, and whether Department of Energy (DOE) user facility proposals are associated with the data. Updates and new features are continually being implemented as user research provides new insights to improve usability, and as standards are updated and improved. 
+The NMDC Submission Portal was designed using a flexible framework leveraging a new modeling approach called the Linked Data Modeling Language (LinkML) and the template-driven spreadsheet tool, [DataHarmonizer](https://github.com/cidgoh/DataHarmonizer). It supports several different community standards, such as the _Minimum Information about any (x) Sequence_ (MIxS) standard from the Genomic Standards Consortium (GSC), the _PROV_ standard for provenance metadata, the _Proteomics Standards Initiative_ (PSI) standards for metaproteomics, and the _Metabolomics Standards Initiative_ (MSI) standards for metabolomics. The NMDC Submission Portal is an intuitive interface that allows researchers to provide information about their study, the metadata about the study’s biosamples, multi-omics data associated with the study, and whether Department of Energy (DOE) User Facility generated data and analysis outputs. Updates and new features are continually being implemented as user research provides new insights to improve usability, and as standards are updated and improved. 
```

Comment:
"The NMDC Submission Portal is an intuitive interface that allows researchers to provide information about their study, the metadata about the study’s biosamples, multi-omics data associated with the study, and whether Department of Energy (DOE) User Facility generated data and analysis outputs."

This sentance is long and ends weird. Suggested update in google doc